Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -9,20 +9,47 @@ public enum ResponseOptions {
     case match(MatchOptions)
 }
 
+/**
+ A `RouteResponse` object is a structure that corresponds to a directions response returned by the Mapbox Directions API.
+ */
 public struct RouteResponse: ForeignMemberContainer {
+    /**
+     Unrecognized properties that are documented as "beta" properties in the Mapbox Directions API response.
+     */
     public var foreignMembers: JSONObject = [:]
     
+    /**
+     The raw HTTP response from the Directions API.
+     */
     public let httpResponse: HTTPURLResponse?
     
+    /**
+     The response identifier used to request the route.
+     */
```

Comment:
I could imagine a scenario where the developer finds it convenient to set this property to an internally generated ID when converting a third-party routing engine’s response to a RouteResponse, but then the navigation SDK would reuse that ID during route refreshing, which could cause an error.

```suggestion
    /**
     The unique identifier that the Mapbox Directions API has assigned to this response.
     */
```